uni-resolver-web/src/main/resources/application.yml

```diff
@@ -265,3 +265,7 @@ uniresolver:
       url: http://driver-did-iscc:8080/
       testIdentifiers:
         - did:iscc:miagwptv4j2z57ci
+    - pattern: "^(did:iid:.+)$"
+        url: http://driver-did-iid:8080/
+        testIdentifiers:
```

Comment:
Pls fix wrong indentation (remove two spaces)

Reply:
@BernhardFuchs sorry for that issue,it's fixed now,thanks